### PR TITLE
Add @ArikaChen's tip for using HTTPS proxy to pull

### PR DESCRIPTION
**Pull request by SvenDowideit**
Docker-DCO-1.1-Signed-off-by: Sven Dowideit SvenDowideit@docker.com (github: SvenDowideit)

Signed-off-by: Sven Dowideit SvenDowideit@docker.com

for #8625

### CODE REVIEW

**Comment by SvenDowideit** [comment]
@ArikaChen @dmp42 @fredlf @jamtur01

File: docs/sources/reference/commandline/cli.md [hunk 1]
```diff
@@ -364,6 +364,20 @@ flag to the Docker daemon as described above.
...
+will be replaced by the proxy's certificates. These need to be added to your Docker
+Host's configuration:
+
+1. Install the `ca-certificates` package for your distribution
+2. Ask your network admin fo the proxy's CA certificate and append them to
+   `/etc/pki/tls/certs/ca-bundle.crt`
+3. Then start your Docker daemon with `HTTPS_PROXY=http://username:password@proxy:port/ docker -d`
+
+This will only add the proxy and authentication to the Docker daemon's requests -
+your `docker builds` and running containers will need extra configuration to use
```
Comment (jamtur01) [inline docs/sources/reference/commandline/cli.md]: s should be outside the formatting \`docker build\`'s

File: docs/sources/reference/commandline/cli.md [hunk 2]
```diff
@@ -364,6 +364,20 @@ flag to the Docker daemon as described above.
...
+### Running a Docker daemon behind a HTTPS_PROXY
+
+When running inside a LAN that uses a `HTTPS` proxy, the Docker Hub certificates
+will be replaced by the proxy's certificates. These need to be added to your Docker
+Host's configuration:
+
+1. Install the `ca-certificates` package for your distribution
+2. Ask your network admin fo the proxy's CA certificate and append them to
+   `/etc/pki/tls/certs/ca-bundle.crt`
+3. Then start your Docker daemon with `HTTPS_PROXY=http://username:password@proxy:port/ docker -d`
```
Comment (xiaods) [inline docs/sources/reference/commandline/cli.md]: I am curious add username and password, Does it is must requirement field?

Comment (thaJeztah) [inline docs/sources/reference/commandline/cli.md]: I *think* that depend on how the proxy is configured. Perhaps a short note that this is just for illustrational purposes and that people should ask their network admin for this as well?

**Comment by thaJeztah** [comment]
Thx for picking this up Sven

File: docs/sources/reference/commandline/cli.md (hunk 2 above)
Comment (xiaods) [inline docs/sources/reference/commandline/cli.md]: I think the hint message is friendly.

**Comment by SvenDowideit** [comment]
@ArikaChen @dmp42 @fredlf @jamtur01

File: docs/sources/reference/commandline/cli.md [hunk 3]
```diff
@@ -364,6 +364,20 @@ flag to the Docker daemon as described above.
...
 as of Docker 1.3.2. It is not recommended to rely on this, as it may change in the future.
 
+### Running a Docker daemon behind a HTTPS_PROXY
+
+When running inside a LAN that uses a `HTTPS` proxy, the Docker Hub certificates
+will be replaced by the proxy's certificates. These need to be added to your Docker
+Host's configuration:
+
+1. Install the `ca-certificates` package for your distribution
+2. Ask your network admin fo the proxy's CA certificate and append them to
```
Comment (jamtur01) [inline docs/sources/reference/commandline/cli.md]: for

File: docs/sources/reference/commandline/cli.md [hunk 4]
```diff
@@ -364,6 +364,20 @@ flag to the Docker daemon as described above.
 Local registries, whose IP address falls in the 127.0.0.0/8 range, are automatically marked as insecure
 as of Docker 1.3.2. It is not recommended to rely on this, as it may change in the future.
 
+### Running a Docker daemon behind a HTTPS_PROXY
+
+When running inside a LAN that uses a `HTTPS` proxy, the Docker Hub certificates
+will be replaced by the proxy's certificates. These need to be added to your Docker
```
Comment (jamtur01) [inline docs/sources/reference/commandline/cli.md]: These certificates...

File: docs/sources/reference/commandline/cli.md [hunk 5]
```diff
@@ -364,6 +364,20 @@ flag to the Docker daemon as described above.
...
+Host's configuration:
+
```
Comment (jamtur01) [inline docs/sources/reference/commandline/cli.md]: host

File: docs/sources/reference/commandline/cli.md [hunk 6]
```diff
@@ -364,6 +364,20 @@ flag to the Docker daemon as described above.
...
+1. Install the `ca-certificates` package for your distribution
+2. Ask your network admin fo the proxy's CA certificate and append them to
+   `/etc/pki/tls/certs/ca-bundle.crt`
+3. Then start your Docker daemon with `HTTPS_PROXY=http://username:password@proxy:port/ docker -d`
+
+This will only add the proxy and authentication to the Docker daemon's requests -
+your `docker build`s and running containers will need extra configuration to use
+the Proxy
```
Comment (jamtur01) [inline docs/sources/reference/commandline/cli.md]: proxy.

File: docs/sources/reference/commandline/cli.md [hunk 7]
```diff
@@ -364,6 +364,22 @@ flag to the Docker daemon as described above.
 Local registries, whose IP address falls in the 127.0.0.0/8 range, are automatically marked as insecure
 as of Docker 1.3.2. It is not recommended to rely on this, as it may change in the future.
 
+### Running a Docker daemon behind a HTTPS_PROXY
+
+When running inside a LAN that uses a `HTTPS` proxy, the Docker Hub certificates
+will be replaced by the proxy's certificates. These certificates need to be added
+to your Docker host's configuration:
```
Comment (fredlf) [inline docs/sources/reference/commandline/cli.md]: Passive voice is a confusing here. Suggest something like "When running inside a LAN that uses a `HTTPS` proxy, you will need to replace the Docker Hub certificates with the proxy's certificates. Follow these steps to configure your Docker host to use the proxy certificates:"

**Comment by fredlf** [comment]
LGTM

**Comment by WuLonghui** [comment]
@thaJeztah @SvenDowideit awesome

**Pull request merged**